src/wallet/rpcnames.cpp

```diff
@@ -823,6 +825,151 @@ name_update ()
 
 /* ************************************************************************** */
 
+RPCHelpMan
+queuerawtransaction ()
+{
+  return RPCHelpMan ("queuerawtransaction",
+      "\nQueue a transaction for future broadcast.",
+      {
+          {"hexstring", RPCArg::Type::STR, RPCArg::Optional::NO, "The hex string of the raw transaction"},
+      },
+      RPCResult {RPCResult::Type::STR_HEX, "", "the transaction ID"},
+      RPCExamples {
+          HelpExampleCli("queuerawtransaction", "txhex") +
+          HelpExampleRpc("queuerawtransaction", "txhex")
+      },
+      [&] (const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
+{
+  std::shared_ptr<CWallet> const wallet = GetWalletForJSONRPCRequest (request);
+  if (!wallet) return NullUniValue;
+  CWallet* const pwallet = wallet.get ();
```

Comment:
Why do you need this?  I think you can just use `wallet` directly below, right?

Reply:
Yes, seems to work.